src/Console/Coverage.php

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
REPOST: Same for `ConsoleShell` inside `src/Shell`. Should we move both into subfolders. Maybe in a different PR?